src/store/main-store/MainActions.ts

```diff
@@ -20,4 +20,8 @@ export default class MainActions extends Actions<
     public changeLanguajeToEnglish(): void {
         this.commit('changeLanguajeToEnglish');
     }
+
+    public activeMaintenancePage(): void {
+        this.commit('activeMaintenancePage');
```

Comment:
@gonzalo-plaza  Please send null parameter to the mutation "this.commit('activeMaintenancePage', **null**)"